scripts/compose/hoprd.cfg.yaml

```diff
@@ -1,5 +1,3 @@
----
-
```

Comment:
The formatter needs to be setup to leave these in. It must support multipage yamls

Reply:
ok, will check

Reply:
Fixed.